Northstar.Client/mod/scripts/vscripts/ui/menu_mod_settings.nut

```diff
@@ -1022,6 +1152,74 @@ void function SendTextPanelChanges( var textPanel )
 					Hud_SetText( textPanel, GetConVarString( c.conVar ) )
 				}
 				break
+			case "color":
+				try {
+					array<string> split = split( newSetting, " " )
+					if ( split.len() < 3 || split.len() > 4)
+					{
+						throw ""
+					}
+
+					array<int> color
+					string clampedNewSetting = ""
+					foreach (string val in split) {
+						int c = int(clamp(val.tointeger(), 0 ,255))
+						color.append(c)
+
+						clampedNewSetting += c + " "
+					}
+
+					if (split.len() == 3)
+					{
+						color.append(255)
+						clampedNewSetting += " 255"
+					}
+
+					Hud_SetColor(colorVGUI, color[0], color[1], color[2], color[3])
+					Hud_SetText( textPanel, clampedNewSetting )
+					SetConVarString( c.conVar, clampedNewSetting )
+
+					// table color = StringToColors( newSetting )
+					// Hud_SetColor(colorVGUI, color.r, color.g, color.b, color.a)
+					// Hud_SetText( textPanel, newSetting )
+					// SetConVarString( c.conVar, newSetting )
+				}
+				catch ( ex )
+				{
+					printt("Failed to send textField change, because:" + ex)
+					ThrowInvalidValue("This setting is a color, and only accepts a four of numbers - you put something we could not parse!\n\n( Use \".\" for the int like \"255 255 255 255\", not \",\". )")
```

Comment:
Is it me or do the instructions here contradict with the example given afterwards?

Reply:
Sorry that I dont get it, what you mean

Reply:
> Sorry that I dont get it, what you mean

Oh, the string.